microsoft-edge/extensions-chromium/index.md

```diff
@@ -102,6 +102,15 @@ To submit your extension to the store, you need to provide the following items:
 
 After you've successfully submitted your extension, your extension undergoes a review process and either passes or fails the certification process.  Owners are notified of the outcome and given next steps as required.  If you submit an extension update to the store, it begins a new review process.
 
+### The Featured badge in Edge Add-ons Store
+
+A **Featured extension** badge in the Microsoft Edge Add-ons Store indicates that the extension has been determined to follow Microsoft's recommended best practices related to security, privacy, transparency, and user experience. The Microsoft Edge Add-ons Store team reviews each extension for adherence to EAS best practices before granting it a **Featured** badge. 
```

Comment:
```suggestion
A **Featured** badge in the Microsoft Edge Add-ons Store indicates that the extension has been determined to follow Microsoft's recommended best practices related to security, privacy, transparency, and user experience. The Microsoft Edge Add-ons Store team reviews each extension for adherence to EAS best practices before granting it a **Featured** badge. 
```
@Bhuvana-Priya - if you send me a url or paste a capture of the badge, I'd be more certain of correct wording.

What does the tooltip say when you hover on a Featured badge?  What does badge look like?  It would be good to show a capture to actually show what a Featured badge looks like.

Reply:
removed "extension": fixed locally in my PR